frontend/src/stores/chatStore.ts

```diff
@@ -0,0 +1,36 @@
+import type { Chat } from '@/types/Chat';
+import { getRequest } from '@/utils/apiRequests';
+import { defineStore } from 'pinia';
+import { useUserStore } from './userStore';
+
+export const useChatStore = defineStore("chats", {
+	//  actions == data definitions
+	state: () => ({
+		allChats: <Chat[]>[],
+	}),
+  // getters == computed values
+	getters: {
+		getMyChats: () => useUserStore().getMe.chats,
+        getAllChats: (state) => state.allChats,
+	},
+  // actions == methods
+	actions: {
+		async refreshAllChats() {
+			try {
+				const data = await getRequest ("chats");
+				this.allChats = data.data;
+			}
+			  catch (e) {
+				console.error(e);
+			}
```

Comment:
Do you know what could go wrong in this try/catch block? 
I'm assuming it will be the request. But what to do in that case? Will we just log an error and that's it? Or is this more a "for later" thing? (if so, maybe we should create at ticket :P)

Reply:
Added an empty array return